deployment/terraform/assets/cluster.tf

```diff
@@ -82,6 +82,47 @@ resource "aws_instance" "app_server" {
   }
 }
 
+data "aws_vpc" "selected" {
+  tags = {
+    Name = "Default VPC"
+  }
+}
+
+data "aws_subnets" "loadtest_subnets" {
+  filter {
+    name = "vpc-id"
+    values = [data.aws_vpc.selected.id]
+  }
+
+  tags = {
+    Name = "loadtest*"
+  }
+}
```

Comment:
Should we make use of the `ClusterSubnetIDs` here if they are defined? Also, we have the variable `cluster_vpc_id`, which is configurable by the user.

Reply:
Ah yeah we should, let me add that.